### DISCUSSION TITLE
blog/2023/vercel-and-netlify

### DISCUSSION THREAD

**Discussion by giscus** [plantree/press-comment#33] [General]
# blog/2023/vercel-and-netlify

https://www.plantree.me/blog/2023/vercel-and-netlify

**Comment by luhuadong** [comment]
文章不错！感觉国内很难发展出类似 Vercel 和 Netlify 的产品。。。

**Reply by plantree** [reply]
develop, preview, deploy. 或许是未来的一种趋势

**Comment by taozhiyuzhuo** [comment]
对于国内来说.中小型公司更愿意花少点的钱去找程序员自己弄ec2来部署服务器啥的.vercel中的功能大部分他们用不到.大公司会有自己的ci/cd体系和工具

**Reply by plantree** [reply]
对于普通开发者，vercel 和 netlify 是个不错的开始，免费，易用，够用。

对于中小型公司，vercel 和 netlify 的成本是有点高了。做一个私有化部署，类似 vercel 和 netlify 的服务，也许能切一部分市场出来。

**Comment by popqq** [comment]
国内有做差不多产品的公司 memfiredb

**Reply by plantree** [reply]
MemFireDB，听名字，更聚焦数据库，而不是静态站点托管。

查了[文档](https://docs.memfiredb.com/docs/app/development_guide/hosting/static-hosting/)，目前对于 CI/CD 和 CDN 加速的支持仍然较弱。

如果只是想部署一套静态网站，Vercel 和 Netlify 应该更容易。

**Reply by taozhiyuzhuo** [reply]
他这个跟vercel是两个产品,他网站出来介绍的第一句话是采用开源的supabase..... 而且文档居然是supabase的截图.

**Comment by kvker** [comment]
在全流程自动化上来说，这俩确实好用。
国内其实一直有一个挺好用的，但是就是不火——LeanCloud，常用的功能这货全有，也有免费的额度。
这玩意儿唯一的大坑，就是免费到付费的差异太大，0元每月直接跳到900+每月。
只能说可惜，好牌打得稀烂。
另外，这货很早就有了，我用了都起码8年了。

**Reply by plantree** [reply]
国内的 SaaS 公司，商业上跑通的并不多。

**Comment by rokerdou** [comment]
个人更喜欢Netlify，我的游戏站 [spotthedifference](https://spotthedifference.pro)就部署在Netlify，可以实现一站式部署，使用vercel的总会有些小问题，构建失败很难定位。

**Comment by McGuffinCN** [comment]
腾讯云出了个EdgeOne Pages，感觉定位和Vercel差不多
